[FrameworkBundle] Remove DI, Entity and Kernel exclusion from the default services file

Jean-Beru opened the pull request:
Since https://github.com/symfony/symfony/pull/59987 and according to the [recipe](https://github.com/symfony/recipes/blob/main/symfony/framework-bundle/7.3/config/services.yaml), the default `config/services.yaml` does not exclude DI, Entity and Kernel file/folders anymore.

Jean-Beru committed:
Update service_container.rst to 7.3

javiereguiluz commented:
Thanks Hubert. This was good, but while merging we did some minor tweaks:

* We removed the `exclude` config in another doc page
* We added an example of using `exclude` because otherwise this is not shown anywhere ... and this feature can still be useful in some cases

See https://github.com/symfony/symfony-docs/commit/59fd3a3f167c8c799fad3256ace24dc8a72501de

Jean-Beru commented:
> Thanks Hubert. This was good, but while merging we did some minor tweaks:
> 
> * We removed the `exclude` config in another doc page
> * We added an example of using `exclude` because otherwise this is not shown anywhere ... and this feature can still be useful in some cases
> 
> See https://github.com/symfony/symfony-docs/commit/59fd3a3f167c8c799fad3256ace24dc8a72501de

Good catch. Thanks @javiereguiluz
